app/views/layouts/_header.html.slim

```diff
@@ -0,0 +1,40 @@
+header class='container-fluid header'
+	#logo_full
+		span class='logo_1' Originate
+		span class='logo_2' Spaces
+	div class='upper_right' id='add_profile'
+		img src=image_path('profile.png') data-toggle="modal" data-target="#profilePopup"
+	div class='upper_right' id='add_post'
+		img src=image_path('plus.png') data-toggle="modal" data-target="#addPostPopup"
+
+// Add Post modal
+#addPostPopup class='modal fade popup' tabindex='-1' role="dialog" aria-labelledby="myModalLabel" aria-hidden="true"
```

Comment:
I agree -- in this case it's useful, since it's a bootstrap modal http://getbootstrap.com/javascript/#modals